### ISSUE TITLE
Support mhtml

### ISSUE THREAD

**Issue by jleclanche** [tht13/html-preview-vscode#10]
Supporting mhtml would be really nice. That'd mean this extension could be used to preview .mhtml and .eml (email) files.

**Comment by AngeloCore** [comment]
Bump

**Comment by alechenken** [comment]
Agreed. The usefulness of mhtml is growing in importance, especially as html documents are becoming more common. The ability of VScode to edit html is invaluable, and should be extended to mhtml as well.

**Comment by thiagojramos** [comment]
It's 2024 and still nothing...
